boefjes/plugins/kat_log4shell/main.py

```diff
@@ -15,30 +15,29 @@
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 logging.basicConfig(level=logging.INFO)
 
-TIMEOUT = 15
```

Comment:
Yes, that could be right, but as it is only used in the `run` function I thought it better to also make it a local variable. That also makes it easier to transfer it to a boefje setting one day. Do you prefer to change it to a setting?

Reply:
I still think this is (and is used as) a constant. When the day comes when this needs to be a dynamic value loaded from settings, then it would be a good time to implement it this way. However, I do get your reasoning. I'll leave it up to you!

Reply:
Okay, I will leave it as is for now then. For the future it may be good to think about how to also inform end-users easily about any constants in a boefje (even if you maybe do not want them to be changed).

Reply:
Sorry, I may have misunderstood you the first time after reading your comment again.

There are plans to have such a feature that allows settings per task (such as timing and other non-environmental settings), but we only have the environment settings for now. Until then the best way to inform users about such constants or side effects would probably be the description in the JSON manifest for the plugin.